Northstar.CustomServers/mod/scripts/vscripts/mp/_stats.nut

```diff
@@ -888,30 +888,32 @@ void function OnWinnerDetermined()
 		player.SetPersistentVar( "kdratio_lifetime_pvp", kdratio_lifetimepvp )
 	}
 
-	// award mvp and top 3 in each team
-	if ( !IsFFAGame() )
+	array<entity> players = GetPlayerArray()
```

Comment:
Code below on this part is just a move from the `_gamestate_mp.nut` without further duplications, also it works on FFA modes as well.